src/components/pages/MissionControl/RenderMissionControl.js

```diff
@@ -61,6 +61,7 @@ const RenderMissionControl = props => {
   useEffect(() => {
     setInstructionText(getMissionControlText(currentStep()));
     setShowModal(true);
+    // eslint-disable-next-line
   }, [hasRead, hasWritten, hasDrawn]);
```

Comment:
I think it's better to just add the currentStep function in the dependency array rather than turn off the linter.

Reply:
Adding it to the dependency array produced the following error

'The 'currentStep' function makes the dependencies of useEffect Hook (at line 65) change on every render. To fix this, wrap the definition of 'currentStep' in its own useCallback() Hook'

As this codebase is being taken off our hands I don't want to go too deep into each of these errors, otherwise I would keep working on this.